### DISCUSSION TITLE
Having trouble with routes in my Astro project using Starlight

### DISCUSSION THREAD

**Discussion by bjorne84** [withastro/starlight#2017] [Q&A] [answered]
Hi, 

I’ve been trying to add Starlight to an existing Astro project and followed the steps over at the [https://starlight.astro.build/manual-setup/ ](https://starlight.astro.build/manual-setup/ ) but I’m stuck. For some reason, the routes aren't picking up content from src/content/docs as they should?

Despite trying different folder structures, such as adding nested docs and wiki folders within src/content/docs (i.e., src/content/docs/docs and src/content/docs/wiki), the routes are not resolving correctly. When attempting to access the routes, it appears to incorrectly match with node_modules/@astrojs/starlight/index.astro instead of the expected content paths.
11:53:14 [WARN] [router] A `getStaticPaths()` route pattern was matched, but no matching static path was found for requested path `/docs`.

Link to my repo
https://github.com/bjorne84/astro-with-starlight

 Does anyone have any ideas what I might be doing wrong or something I overlooked?

Thanks a ton for any help you can give!

**Comment by Fryuni** [comment, marked as answer]
Hey @bjorne84, I'm not sure what exactly is the problem you are facing... Your repo is working fine.

Your `docs` collection has the following structure:
- `docs`
  - `test2.md` -- Defines the route `/docs/test2`
- `wiki`
  - `test.md` -- Defines the route `/wiki/test`

Those two routes are working as intended.

The error message you sent about there being no match for `/docs` is correct. Your project doesn't define any `/docs` route. Neither with a custom page nor using Starlight.

**Reply by Fryuni** [reply]
![image](https://github.com/withastro/starlight/assets/11063910/9c9224fa-a2eb-4850-b2f3-e8af12c6640c)

**Comment by bjorne84** [comment]
thanks, you are right
